examples/units_and_coordinates/getting_observer_location.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+========================================
+Getting the observer location from a Map
+========================================
+
+How to access the observer location from a Map and interpret it.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+from astropy.coordinates import get_body
+from astropy.constants import R_earth
+
+from sunpy.data.sample import AIA_171_IMAGE
+import sunpy.map
+
+###############################################################################
+# We start with the sample data
+aiamap = sunpy.map.Map(AIA_171_IMAGE)
+
+###############################################################################
+# You can access the observer coordinate with:
+print(aiamap.observer_coordinate)
+
+###############################################################################
+# This provides the location of the SDO as defined in the header and is
+# necessary to fully define the helioprojective coordinate system which
+# depends on where the observer is. Let's see where this is with respect
+# to Earth. SDO is a geosynchronous orbit with a semi-major axis of
+# 42,164.71 km and an inclination of 28.05 deg.
+# We will convert it to Geocentric Celestial Reference System (GCRS)
+# whose center is at the Earth's  center-of-mass.
+sdo_gcrs = aiamap.observer_coordinate.transform_to('gcrs')
+sun = get_body('sun', aiamap.date)
```

Comment:
Don't use Astropy's `get_body()` because it bakes in assumptions.  While those assumptions happen to be essentially fine in this case, an instructive example should stick with our `get_body_heliographic_stonyhurst()`.

Reply:
What assumptions?

Reply:
It corrects for light travel time for an observer at Earth center.  Here, SDO isn't that far from Earth center to significantly change the light travel time, and the Sun doesn't move significantly during the light travel time anyway, so it's essentially moot.  But, in general we'd tell people to use `get_body_heliographic_stonyhurst()`, so that's what they should use here too.

Reply:
Okie dokie!